apps/clock/test/marionette/lib/actions.js

```diff
@@ -13,9 +13,9 @@ function ClockAppActions(options) {
         // we need to disable the keyboard to avoid intermittent failures on
         // Travis (transitions might take longer to run and block UI).
         'dom.mozInputMethod.enabled': false,
-        // Do not require the B2G-desktop app window to have focus (as per the
-        // system window manager) in order for it to do focus-related things.
-        'focusmanager.testmode': true
+        // NOTE: We used to require 'focusmanager.testmode=true' here
+        // back in the B2G-Desktop days, but that breaks on-device testing.
```

Comment:
Might as well change this to 'NOTE: We used to require 'focusmanager.testmode=true' for B2G Desktop but Mulet and Device tests do not require it.'

We'll probably want to accompany landing this with a quick email to dev-fxos to mention that your mileage will be reduced when running tests on b2g-desktop (possibly) because of this change.